src/Compilers/Core/Portable/Operations/IDynamicInvocationExpression.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Semantics
+{
+    /// <summary>
+    /// Represents a dynamically bound invocation expression.
+    /// </summary>
+    /// <remarks>
+    /// This interface is reserved for implementation by its associated APIs. We reserve the right to
+    /// change it in the future.
+    /// </remarks>
+    public interface IDynamicInvocationExpression : IHasDynamicArgumentsExpression
```

Comment:
>IDynamicInvocationExpression [](start = 21, length = 28)

Should we expose information about type arguments?

Reply:
Type arguments are stored neither on `BoundDynamicInvocation` node in C# nor on `BoundLateInvocation` node in VB. A dynamic invocation generally has the dynamic member reference as it's Expression child, which already exposes the type arguments.